environment.yml

```diff
@@ -14,7 +14,13 @@ dependencies:
   - scikit-image
   - matplotlib
   - seaborn
+  - jupyterlab-lsp
+  - python-lsp-server[all]
+  - jupyterlab-git
+  - jupyterlab_execute_time
   - pip:
-    - alpineer==0.1.6
-    - pyFlowSOM==0.1.14
+    - jupyterlab-skip-traceback
+    - lckr-jupyterlab-variableinspector
+    - alpineer==0.1.7
```

Comment:
Should we bump alpineer to 0.1.8 instead? [https://github.com/angelolab/alpineer/releases/tag/v0.1.8](https://github.com/angelolab/alpineer/releases/tag/v0.1.8)